ci/helpers/albanode.py

```diff
@@ -90,3 +96,23 @@ def _map_node_disks(albanode):
             # Map aliases to the diskname
             mapping[diskname] = disk['aliases']
         return mapping
+
+    @staticmethod
+    def get_disks_from_asdmanager(ip, port, username, password):
```

Comment:
You could use from ci.helpers.asdmanager import ASDManagerClient
ASDManagerClient(albanode)
disks = asd_client.get_disks()